src/library/ResourceUtils/ResourceUtils.ts

```diff
@@ -70,6 +70,12 @@ export class ResourceUtils {
    * @returns array of elements found at the provided path
    */
   getValuesAtResourcePath(resource: any, elementPath: string): any[] {
+    const pathSections = elementPath.split('.');
+    if (!resource || (resource.resourceType !== pathSections[0])) return [];
+    return this.getValuesAtResourcePathInner(resource, elementPath);
+  }
+
+  private getValuesAtResourcePathInner(resource: any, elementPath: string): any[] {
     const pathSections = elementPath.split(".");
```

Comment:
you're splitting the `elementPath` twice. either send the pathSections in as a parameter or don't the split in the `getValuesAtResourcePath`.